app/_locales/en/messages.json

```
@@ -1078,7 +1078,7 @@
     "message": "Backup gas price is provided as the main gas estimation service is unavailable right now."
   },
   "eth_accounts": {
-    "message": "See address, account balance, activity and initiate transactions",
+    "message": "See address, account balance, activity and suggest transactions",
```

One alternative option is to just drop the part about showing confirmations until we have wording we are sure works well here (we didn't mention it previously).
```suggestion
    "message": "See address, account balance, and activity.",
```